modules/xfem/src/materials/LevelSetMultiMaterial.C

```diff
@@ -0,0 +1,159 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "LevelSetMultiMaterial.h"
+#include "AuxiliarySystem.h"
```

Comment:
If level set is a nonlinear variable, do you think if this code would also work?

Reply:
It should work. I removed this include line.